app/scripts/modules/core/delivery/executionGroup/execution/execution.less

```diff
@@ -1,6 +1,6 @@
 @import "../../../presentation/less/imports/commonImports.less";
 
-execution {
+.execution {
```

Comment:
react doesn't render an <execution> component anymore, so have to use classes now.